InterviewPresenter.swift

```diff
@@ -0,0 +1,51 @@
+
+import UIKit
+import Foundation
+
+protocol ProfileViewInput: AnyObject {
+    var output: ProfileViewOutput? { get set }
+    
+    func setResult(value: String)
+}
+
+protocol ProfileViewOutput {
+    func viewIsReady()
+}
+
+enum TypeCalculate: String {
+    case sum
+    case difference
+}
+
+class Presenter: ProfileViewOutput {
+    weak var view: ProfileViewInput!
+    
+    var a = Int()
+    let b: Int
+    var c = 0
+    let typeCalculate: TypeCalculate
+    
+    init(koef a: Int, b: Int, c: Int, typeCalculate: TypeCalculate) {
+        a = a
+        b = b
```

Comment:
should be compile error here
self.a = a
self.b = b
self.c = c

a lot of compile errors in PR, check in Xcode for errors before PR and fix it please